Lib/fontParts/base/lib.py

```diff
@@ -355,10 +361,10 @@ def items(self) -> List[Tuple[str, LibValueType]]:
         """
         return super(BaseLib, self).items()
 
-    def keys(self) -> List[str]:
-        """Return an unordered list of the lib's keys.
+    def keys(self) -> BaseKeys[str]:
+        """view the lib's keys.
```

Comment:
I think that the term `view` here is going to be confusing. Would say "Returns the lib's keys". Does that make sense?

Reply:
Agreed! I just got hung up on the repetitiveness of `Return the lib's ...` and `:return: The lib's ...`.